internal/npm_install/npm_install.bzl

```diff
@@ -59,6 +64,10 @@ error.
 """,
         default = False,
     ),
+    "args": attr.string_list(
+        doc = """Arguments to passed to the package manager.""",
```

Comment:
give a link to their CLI doc so I can find what args are allowed
(might require that you move the attribute to appear under both yarn/npm versions)

Reply:
👍